applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -27,6 +27,73 @@
  * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration
  * Store.
  *
+ * <p>
+ *     Azure Configuration Store operations allowed by the client are adding, retrieving, updating, and deleting
+ *     ConfigurationSettings, additionally more complex operations of listing settings or revision of a setting based
+ *     on a filter are supported as well.
+ * </p>
```

Comment:
Just a general formatting preference - Put 'Azure' directly after the opening `<p>` element, and similarly, put the closing element at the end of the sentence rather than on a new line. Then, don't do any indentation like you have here - just a single space between the asterisk and the text.